src/common/xBuild.ts

```diff
@@ -6,9 +6,11 @@ type XmlElement = {
   [tag: string]: any;
 };
 
-const nsMap = mapKeys(namespaces, (v, k) => {
-  return `@xmlns:${k}`;
-});
+const nsMap = function() {
+  return mapKeys(namespaces, (v, k) => {
+    return `@xmlns:${k}`;
+  });
+}
```

Comment:
```suggestion
};
```
Missing semicolon leads to build error
```
  13:2  error  Missing semicolon  @typescript-eslint/semi
```

Reply:
:facepalm: That's what I get for thinking "hey, I have 5 min to spare. Let's fix this real quick"

Thanks for noticing.